lib/python/Screens/TimerEdit.py

```diff
@@ -38,80 +39,82 @@ class TimerEditList(Screen):
 
 	def __init__(self, session):
 		Screen.__init__(self, session)
-		Screen.setTitle(self, _("Timer List"))
-
+		self.setTitle(_("Timer List"))
 		self.onChangedEntry = []
-		list = []
-		self.list = list
+		self.list = []
 		self.fillTimerList()
-
-		self["timerlist"] = TimerList(list)
-
+		self["timerlist"] = TimerList([])
 		self.key_red_choice = self.EMPTY
 		self.key_yellow_choice = self.EMPTY
 		self.key_blue_choice = self.EMPTY
-
-		self["key_red"] = Button(" ")
-		self["key_green"] = Button(_("Add"))
-		self["key_yellow"] = Button(" ")
-		self["key_blue"] = Button(" ")
-
+		self["key_red"] = StaticText("")
+		self["key_green"] = StaticText(_("Add"))
+		self["key_yellow"] = StaticText("")
+		self["key_blue"] = StaticText("")
 		self["description"] = Label()
 		self["ServiceEvent"] = ServiceEvent()
 		self["Event"] = Event()
-
-		self["actions"] = ActionMap(["OkCancelActions", "DirectionActions", "ShortcutActions", "TimerEditActions"],
-			{
-				"ok": self.openEdit,
-				"cancel": self.leave,
-				"green": self.addCurrentTimer,
-				"log": self.showLog,
-				"left": self.left,
-				"right": self.right,
-				"up": self.up,
-				"down": self.down
-			}, -1)
+		self["actions"] = ActionMap(["OkCancelActions", "DirectionActions", "ShortcutActions", "TimerEditActions"], {
+			"ok": self.openEdit,
+			"cancel": self.leave,
+			"green": self.addCurrentTimer,
+			"log": self.showLog,
+			# "top": self.moveTop,
+			"left": self.pageUp,
+			"up": self.moveUp,
+			"down": self.moveDown,
+			"right": self.pageDown,
+			# "bottom": self.moveBottom
+		}, prio=-1)
 		self.setTitle(_("Timer overview"))
```

Comment:
Or is this the correct title?  If so, it should be "Timer Overview" and the earlier setTitle() should be removed.